packages/web-runtime/themes/owncloud/theme.json

```diff
@@ -118,7 +118,7 @@
         "swatch-warning-default": "",
         "swatch-warning-hover": "",
         "swatch-warning-muted": "",
-        "text-default": "#c2c2c2",
+        "text-default": "#DADCDF",
```

Comment:
we usually have lower case chars for color hex codes. of course not a technical issue at all, but not having it consistent makes me cringe a little bit 😁